src/engraving/layout/dev/tlayout.cpp

```diff
@@ -643,75 +643,78 @@ void TLayout::layoutArticulation(Articulation* item, LayoutContext&)
     item->setLayoutData(data);
 }
 
-void TLayout::layout(BagpipeEmbellishment* item, LayoutContext& ctx)
+static void layoutBagpipeEmbellishment(const BagpipeEmbellishment* item, const LayoutContext& ctx, BagpipeEmbellishment::LayoutData& data)
```

Comment:
I'm not sure if this will ever be called. I believe BagpipeEmbellishment is a strange kind of element that is only used in the palettes; when you apply it to a score, it just becomes a group of grace notes. Not 100% sure though.